docs/logging.md

```diff
@@ -0,0 +1,40 @@
+# Logging in Odo
+
+[Glog](https://godoc.org/github.com/golang/glog) is used for V style logging in Odo.
+
+
+## Working
+
+Every ODO command takes an optional flag `-v` that must be used with an integer log level in the range from 0-9. And any INFO severity log statement that is logged at a level lesser than or equal to the one passed with the command alongside `-v` flag will be logged to the STDOUT.
+
+All ERROR severity level log messages will always be logged immaterial of the passed `v` level.
+
+
+## Usage
+
+Every source file that needs to log messages will need to import glog as under:
+
+```
```

Comment:
make it ```go

not ```

so there is formatting in markdown

Reply:
Did you mean get ``` and the statement in same line ?
If so, done